patches/server/0110-Ensure-correct-attack-damage-on-slot-change.patch

```diff
@@ -0,0 +1,88 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: necrozma <necrozma999@gmail.com>
+Date: Fri, 14 Jul 2023 22:16:17 -0600
+Subject: [PATCH] Ensure correct attack damage on slot change
```

Comment:
If I remember correctly @ThatApplePieGuy closed the original pullrequest because the patch didn't fix the problem 100%